pkg/splunk/client/awss3client.go

```diff
@@ -249,7 +249,7 @@ func (awsclient *AWSS3Client) DownloadApp(ctx context.Context, remoteFile, local
 			IfMatch: aws.String(etag),
 		})
 	if err != nil {
-		scopedLog.Error(err, "Unable to download item %s, %v", remoteFile, err)
+		scopedLog.Error(err, fmt.Sprintf("Unable to download item %s, %v", remoteFile, err))
```

Comment:
The `err` is already included in the log message so we can probably remove it from the `fmt.Sprintf()`.

Reply:
Good point. Resolved in the last commit.